From a71adcc243ac6263efa23390ca8bb82ae4841202 Mon Sep 17 00:00:00 2001
From: James Prestwood <james.prestwood@linux.intel.com>
Date: Fri, 5 Apr 2019 15:32:54 -0700
Subject: netdev: skip associate event when not in OWE/FT

The associate event is only important for OWE and FT. If neither of
these conditions (or FT initial association) are happening we do
not need to continue further processing the associate event.
---
 src/netdev.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/netdev.c b/src/netdev.c
index 661fb81..849ab2b 100644
--- a/src/netdev.c
+++ b/src/netdev.c
@@ -2389,6 +2389,9 @@ static void netdev_associate_event(struct l_genl_msg *msg,
 	if (netdev->aborting)
 		return;
 
+	if (!netdev->owe && !netdev->in_ft && !netdev->handshake->mde)
+		return;
+
 	if (!l_genl_attr_init(&attr, msg)) {
 		l_debug("attr init failed");
 		return;
-- 
cgit 1.2-0.3.lf.el7

